yarn-project/pxe/src/simulator_oracle/simulator_oracle.test.ts

```diff
@@ -544,32 +566,7 @@ describe('Simulator oracle', () => {
       );
       return taggedLogs;
     }
-
-    it('should store an incoming note that belongs to us', async () => {
```

Comment:
Now the tests test less, but that is fine as the complexity has moved to the contract out of PXE.

Instead of testing that the note has actually been added to the db, I just test that a process_log func has been called on the contract.

Given the above I felt like it made not sense to have these 3 tests anymore so I reduced them to only 1.